libraries/c_sdk/standard/mqtt/src/iot_mqtt_serializer_deserializer_wrapper.c

```diff
@@ -0,0 +1,772 @@
+/*
+ * FreeRTOS MQTT V2.1.1
```

Comment:
Please update all the headers to indicate this is the Shim, not MQTT library. This can be done as part of a different PR.